utils/format_unifier.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+from pptx import Presentation
+from PyPDF2 import PdfFileReader
+from tika import parser
+from utils.tika_client import TikaClient
+import os
+
+
+class FormatUnifier:
+    def __init__(self) -> None:
+        tika_hist = os.environ['TIKA_HOST']
+        tika_port = os.environ['TIKA_PORT']
+        self._tika_client = TikaClient(tika_hist, tika_port)
+
+    def extract_text(self, path_to_file: str) -> str:
+        path = Path(path_to_file)
+        if not path.exists():
+            raise FileNotFoundError
+        if path.suffix == '.pdf':
```

Comment:
To  można zrobić sensowniej (zostawimy póki co tak, ale możemy pamiętać). Tika ma swój endpoint do wykrycia typu pliku. Czyli nawet jak dasz nazwę `dubstep.mp3`, a jest to tak naprawdę pdf, to Tika to wykryje